packages/SwingSet/src/vats/vat-timerWrapper.js

```diff
@@ -41,6 +42,29 @@ export function buildRootObject(vatPowers) {
         repeaters.set(index, vatRepeater);
         return vatRepeater;
       },
+      createNotifier(delaySecs, interval) {
+        assert(
+          Nat(delaySecs) >= 0,
+          details`createNotifier's first parameter must be a non-negative integer. ${delaySecs}`,
+        );
+        assert(
+          Nat(interval),
+          details`createNotifier's second parameter must be an integer, ${interval}`,
+        );
+
+        const index = D(timerNode).createRepeater(delaySecs, interval);
+        console.log(`TIMER: index: ${index}`);
+        const { notifier, updater } = makeNotifierKit();
+        const updateHandler = harden({
+          wake: timestamp => {
+            console.log(`TimerNotifier tick ${timestamp}`);
+            updater.updateState(timestamp);
+          },
```

Comment:
Assuming that the `console` call is a vestige (see previous comment) how about
```suggestion
          wake: updater.updateState,
```

Reply:
done.